drag_and_drop_v2/public/js/view.js

```diff
@@ -0,0 +1,258 @@
+                        'tabindex': 0,
+                        'dropzone': 'move',
+                        'aria-dropeffect': 'move',
+                        'data-zone': zone.title
+                    },
+                    style: {
+                        top: zone.y_percent + '%', left: zone.x_percent + "%",
+                        width: zone.width_percent + '%', height: zone.height_percent + "%",
+                    }
+                },
+                [
+                    h('p', { className: className }, zone.title),
+                    h('p', { className: 'zone-description sr' }, zone.description)
+                ]
+            )
+        );
+    };
+
+    var feedbackTemplate = function(ctx) {
+        var feedback_display = ctx.feedback_html ? 'block' : 'none';
+        var reset_button_display = ctx.display_reset_button ? 'block' : 'none';
+        var properties = { attributes: { 'aria-live': 'polite' } };
+        return (
+            h('section.feedback', properties, [
+                h(
+                    'a.reset-button',
+                    { style: { display: reset_button_display }, attributes: { tabindex: 0 }},
+                    gettext('Reset exercise')
+                ),
+                h('h3.title1', { style: { display: feedback_display } }, gettext('Feedback')),
+                h('p.message', { style: { display: feedback_display }, innerHTML: ctx.feedback_html })
+            ])
+        );
+    };
+
+    var keyboardHelpTemplate = function(ctx) {
+        var dialog_attributes = { role: 'dialog', 'aria-labelledby': 'modal-window-title' };
+        var dialog_style = {};
+        return (
+            h('section.keyboard-help', [
+                h('a.keyboard-help-button', { attributes: { tabindex: 0 } }, gettext('Keyboard Help')),
+                h('div.keyboard-help-dialog', [
+                    h('div.modal-window-overlay'),
+                    h('div.modal-window', { attributes: dialog_attributes, style: dialog_style }, [
+                        h('div.modal-header', [
+                            h('h2.modal-window-title', gettext('Keyboard Help'))
+                        ]),
+                        h('div.modal-content', [
+                            h('p', gettext('You can complete this exercise using only your keyboard.')),
+                            h('ul', [
+                                h('li', gettext('Use "Tab" and "Shift-Tab" to navigate between items and zones.')),
+                                h('li', gettext('Press "Enter", "Space", "Ctrl-m", or "⌘-m" on an item to select it for dropping, then navigate to the zone you want to drop it on.')),
+                                h('li', gettext('Press "Enter", "Space", "Ctrl-m", or "⌘-m" to drop the item on the current zone.')),
+                                h('li', gettext('Press "Escape" if you want to cancel the drop operation (e.g. because you would like to select a different item).')),
```

Comment:
'Press "esc" if you want to cancel the drop operation (for example, to select a different item).'